sources/platform/integrations/ai/mcp.md

```diff
@@ -152,27 +152,58 @@ By default, the main Actors MCP Server starts with a single default [RAG Web Bro
+The following tool categories are available:
 
-:::note Optional tools
+- _Actor discovery and management_ (default, always enabled): Search for [Actors](https://docs.apify.com/platform/actors) (`search-actors`), view details (`get-actor-details`), and dynamically add them (`add-actor`).
+- _docs_ (default, can be disabled): Search Apify documentation (`search-apify-docs`) and fetch specific documents (`fetch-apify-docs`).
+- _runs_ (optional): Get a list of your [Actor runs](https://docs.apify.com/platform/actors/running/runs-and-builds#runs) (`get-actor-run-list`), specific run details (`get-actor-run`), and logs from a specific Actor run (`get-actor-log`).
+- _storage_ (optional): Access [datasets](https://docs.apify.com/platform/storage/dataset) and [key-value stores](https://docs.apify.com/platform/storage/key-value-store), including their records (`get-dataset`, `get-dataset-items`, `get-dataset-list`, `get-key-value-store`, `get-key-value-store-keys`, `get-key-value-store-record`, `get-key-value-store-records`).
+- _preview_ (optional): Experimental tools in preview mode. Call any Actor using API (`call-actor`).
 
-Helper tool categories marked with (*) are not enabled by default in the MCP server and must be explicitly enabled using the `tools` argument (either the `--tools` command line argument for the stdio server or the `?tools` URL query parameter for the remote MCP server). The `tools` argument is a comma-separated list of categories with the following possible values:
+The _Actor discovery and management_ tools are always present and cannot be disabled.
+The _docs_ tools are enabled by default but can be switched off using the `tools` parameter.
 
-- `docs`: Search and fetch Apify documentation.
-- `runs`: Get Actor runs list, run details, and logs from a specific Actor run.
-- `storage`: Access datasets, key-value stores, and their records.
-- `preview`: Experimental tools in preview mode.
+### Configure mcp.apify.com using query parameters
 
-:::
+Use the `tools` query parameter to enable or disable specific tool categories.
+
+For example, to enable only the `runs` and `storage` tools, you can use:
+
+```text
+https://mcp.apify.com/?tools=runs,storage
+```
+
+The server will expose all `Actor discovery and management tools`, as well as `runs` and `storage`.
+
+### Configure STDIO server using CLI arguments
+
+When running the MCP server via the command line, you can specify the tools using the `--tools` parameter.
+For example, to enable only the `runs` and `storage` tools, you can run:
+
+```bash
+npx @apify/actors-mcp-server --tools runs,storage
+```
+
+## Dynamic Actor tooling
+
+One of the powerful features of MCP with Apify is **dynamic Actor tooling** – the ability for an AI agent to find new tools (Actors) as needed and incorporate them.
```

Comment:
"Dynamic Actor tooling" ... IMO the standard name is "dynamic tool discovery" so this could be "Dynamic discovery of Actors"